examples/python/route_guide/run_codegen.py

```diff
@@ -28,5 +27,16 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Runs the protoc with gRPC plugin to generate protocol messages and gRPC stubs.
-python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/route_guide.proto
```

Comment:
There is nothing *incorrect*, but I think the concern is that we've decided to mix-and-match Shell-anese in our United States of Python-published "Baby's First Services" book. It makes sense to e.g. 1) provide a unified front of, "This is the one way to do it if you're just interested in running," or 2) explicitly state "These are the different ways to do it," or somesuch, with there existing arguments that 1 is superior to 2 (I don't have a strong stance here, though).